src/Nethermind/Nethermind.AuRa.Test/Reward/AuRaRewardCalculatorTests.cs

```diff
@@ -112,11 +112,11 @@ public void calculates_rewards_correctly_after_contract_transition(long blockNum
         public void calculates_rewards_correctly_for_ommers(long blockNumber, long expectedReward)
         {
             _block.Header.Number = blockNumber;
-            _block.Body.Ommers = new[]
+            _block.Body = new BlockBody(_block.Body.Transactions, new[]
```

Comment:
Why is constructing new BlockBody needed?

Reply:
making it immutable - we use Body.Empty and we  modified static Body.Empty which broke tests